artifactory/commands/transferfiles/transfer.go

```diff
@@ -198,6 +198,13 @@ func (tdc *TransferFilesCommand) transferSingleRepo(sourceRepoKey string, target
 			return
 		}
 	}
+	// Performing a reset on the source if chunks were not removed
+	runningNodes, err := getRunningNodes(tdc.sourceServerDetails)
+	if err != nil {
+		log.Error(err)
+	} else {
+		stopTransferOnArtifactoryNodes(srcUpService, runningNodes)
+	}
```

Comment:
Notice the above code is duplicated twice in the code.
We should theregore move into a util function named `stopTransferInArtifactory` which recieves no arguments. This utility function should return an error, which is checked here. amd also in the other functions that will use this utility function.

Also, the existing `stopTransferOnArtifactoryNodes` should be renamed to `stopTransferInArtifactoryNodes`.

I also suggest running this lceanup before initiating a new phase (any new phase) and not after finishing all phases for a repository.

Reply:
Done, but the function must get the serverDetails and the srcUpService as arguments